bokeh/core/_templates/autoload_js.js

```diff
@@ -83,23 +105,26 @@ calls it with the rendered model.
   }
   {%- endif %}
 
+  function inject_raw_css(css) {
+    const element = document.createElement("style");
+    element.appendChild(document.createTextNode(css));
+    document.body.appendChild(element);
+  }
+
   var js_urls = {{ js_urls|json }};
+  var css_urls = {{ css_urls|json }};
 
   var inline_js = [
-    {%- for js in js_raw %}
-    function(Bokeh) {
-      {{ js|indent(6) }}
-    },
-    {% endfor -%}
     function(Bokeh) {
-      {%- for url in css_urls %}
-      console.log("Bokeh: injecting CSS: {{ url }}");
-      Bokeh.embed.inject_css({{ url|json }});
-      {%- endfor %}
       {%- for css in css_raw %}
-      console.log("Bokeh: injecting raw CSS");
-      Bokeh.embed.inject_raw_css({{ css }});
+        console.log("Bokeh: injecting raw CSS");
```

Comment:
As long as we are editing this file can we remove the log statements, or lower them to `debug`